js/blocks.js

```diff
@@ -3484,6 +3484,9 @@ function Blocks(canvas, stage, refreshCanvas, trashcan, updateStage, getStageSca
         if (['start', 'action', 'nameddo', 'namedarg', 'nameddoArg', 'calc', 'calcArg', 'namedcalcArg', 'storein'].indexOf(name) != -1) {
             this.checkPaletteEntries(name);
         }
+        console.log("Finished block loading");
+        var myCustomEvent = new Event('finishedLoading');
+        document.dispatchEvent(myCustomEvent);
```

Comment:
This is the correct idea, but there is a problem since loadNewBlocks is called every time a stack of blocks is loaded, not just on first load. Unless the event handler is disabled, it will trigger multiple time.